tests/CMakeLists.txt

```diff
@@ -8,10 +8,27 @@ if(NOT MSVC)
 		"-Wall" "-pedantic-errors" "-Wold-style-cast" "-fno-rtti")
 endif()
 
+# Enable vcpkg packages.
+# For some reason, they don't get detected automatically in this multi-level config.
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake" CACHE STRING "Vcpkg toolchain file")
+if(UNIX AND NOT APPLE)
+	set(CUSTOM_PATH "vcpkg_installed/x64-linux")
+elseif(UNIX AND APPLE)
+	set(CUSTOM_PATH "vcpkg_installed/${VCPKG_TARGET_TRIPLET}")
+else()
+	if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+		set(CUSTOM_PATH "build/ci/vcpkg_installed/x64-windows-release")
+	else()
+		set(CUSTOM_PATH "build/ci/vcpkg_installed/win64-release")
+	endif()
+endif()
+set(CMAKE_PREFIX_PATH "${CMAKE_SOURCE_DIR}/${CUSTOM_PATH}")
```

Comment:
Probably just my lack of understanding, but why do we need to specify a tool-chain file and custom paths here?

Reply:
I'm honestly not sure. It's supposed to "just work", but it didn't, and this was the solution recommended to me... I'm not clear on why it didn't import the packages correctly by default.

Reply:
I feel we need to understand this a little bit better than we currently do. CMake has very smart caching systems and if this somehow affects the caching systems, then we will lose quite some efficiency benefits from CMake.

Reply:
I think it's fixed now.